assets/csidriveroperators/gcp-pd/07_deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gcp-pd-csi-driver-operator
+  namespace: openshift-cluster-csi-drivers
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: gcp-pd-csi-driver-operator
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        name: gcp-pd-csi-driver-operator
+    spec:
+      containers:
+      - args:
+        - start
+        env:
+        - name: DRIVER_IMAGE
+          value: quay.io/openshift/origin-gcp-pd-csi-driver:latest
```

Comment:
Is there any reason for not using the placeholders (e.g., `${DRIVER_IMAGE}`) now?

Reply:
There isn't. This has been updated.